Kitodo/src/main/java/de/sub/goobi/helper/servletfilter/RequestControlFilter.java

```diff
@@ -73,7 +73,7 @@
      */
     @Override
     @SuppressWarnings({"unchecked", "rawtypes" })
-    public void init(FilterConfig config) throws ServletException {
+    public void init(FilterConfig config) {
```

Comment:
Is this correct removement as there is a suppressed warnings annotation?

Reply:
Those warnings are not suppressing exception. If this exception would be necessary there IDE would inform about it as an error, not warning.

Reply:
Ok, then is this change ok.